xml/ftp_banners.xml

```diff
@@ -184,11 +184,21 @@ against these patterns to fingerprint FTP servers.
     <param pos="0" name="service.family" value="ProFTPD"/>
     <param pos="0" name="service.product" value="ProFTPD"/>
   </fingerprint>
-  <fingerprint pattern="^-{10} Welcome to Pure-FTPd (.*)-{10}$">
+  <fingerprint pattern="^=\(&lt;\*&gt;\)=\-\.:\. \(\( Welcome to Pure-FTPd ([\d\.]+) \)\) \.:\.\-=\(&lt;\*&gt;\)=\-$">
```

Comment:
Paul's comment about `.` was for `([\d\.]+)`, which is correct.  Fixed on both accounts (nice catch).